go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/zorchenhimer/MovieNight
 
-go 1.12
+go 1.13
```

Comment:
Why is this versionbump necessary?

Reply:
It's not even because of something I changed, my IDE spat out this complaint:
`signed shift count 60 (untyped int constant) requires go1.13 or later`, referring to settings.go, line 173